Sources/AccioKit/Commands/Protocols/DependencyInstaller.swift

```diff
@@ -34,9 +34,16 @@ extension DependencyInstaller {
                 let frameworkCheckoutPath: String = checkoutsDirUrl.appendingPathComponent(fileName).path
 
                 if try FileManager.default.isDirectory(atPath: frameworkCheckoutPath) {
-                    try bash("git -C '\(frameworkCheckoutPath)' reset HEAD --hard --quiet 2> /dev/null")
-                    try bash("git -C '\(frameworkCheckoutPath)' clean -fd --quiet 2> /dev/null")
-                    try bash("git -C '\(frameworkCheckoutPath)' clean -fdX --quiet 2> /dev/null")
+                    do {
+                        try bash("git -C '\(frameworkCheckoutPath)' reset HEAD --hard --quiet 2> /dev/null")
+                        try bash("git -C '\(frameworkCheckoutPath)' clean -fd --quiet 2> /dev/null")
+                        try bash("git -C '\(frameworkCheckoutPath)' clean -fdX --quiet 2> /dev/null")
+                    }
+                    catch {
+                        // Remove checkout if git reset fails for some reason
+                        // If the checkout is missing, SPM will automatically clone again in the next step
+                        FileManager.default.removeItem(atPath: frameworkCheckoutPath)
```

Comment:
Will this line do the same thing as `bash("rm -rf \(frameworkCheckoutPath)")`? And if so, what's the pro and cons of using `removeItem` instead of `rm -rf`? I'm guessing that if `removeItem` actually does the same thing, then it might be faster since it's not going the detour of bash and the `rm` command but directly does what's needed. In that case, we could replace all `rm -rf` calls with `removeItem`?

Reply:
Good idea to investigate this! Will look into it 👍

Reply:
I opened up #41 to discuss this and will use the existing mechanism for this PR